scripts/run-ci.sh

```diff
@@ -32,6 +32,12 @@ if [ ! -z "$CPPLINT" ]; then
     exit 0
 fi
 
+if [ ! -z "$GOFMT" ]; then
```

Comment:
Please reuse the existing variable `CPPLINT` to determine if `gofmt.sh` should be run.
Also `ABORT_ON_ERR` should set before calling so the CI job fails on formatting errors.